### ISSUE TITLE
UserCreatorWizard -> does not correctly manage login/password

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Risehat/freemedforms#246]
```
We have to find a workflow to correctly manage the user login and password.
```

Original issue reported on code.google.com by `eric.mae...@gmail.com` on 19 Dec 2012 at 9:40

**Comment by GoogleCodeExporter** [comment]
```
"manage" - you mean display? technical problems or UI behaviour?
```

Original comment by `christian.a.reiter@gmail.com` on 19 Dec 2012 at 9:42

**Comment by GoogleCodeExporter** [comment]
```
No I mean, when page is activated:
- add the login
- what for the password (is not stored in clear)
  * add a button "change password"
...
```

Original comment by `eric.mae...@gmail.com` on 19 Dec 2012 at 10:04

**Comment by GoogleCodeExporter** [comment]
```
ah ok.
for the login: simple.
for the password: leave it empty with a PlaceHolderText "Enter a new password 
to change it"? (in both fields)
```

Original comment by `christian.a.reiter@gmail.com` on 19 Dec 2012 at 10:52

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision fc41b1177013.
```

Original comment by `eric.mae...@gmail.com` on 22 Jan 2013 at 9:51
- Changed state: **Fixed**
